tests/integration/helpers/ticket-attendees-test.js

```diff
@@ -0,0 +1,15 @@
+import { module, test } from 'qunit';
+import { setupIntegrationTest } from 'open-event-frontend/tests/helpers/setup-integration-test';
+import hbs from 'htmlbars-inline-precompile';
+import { render } from '@ember/test-helpers';
+
+module('Integration | Helper | ticket-attendees', function(hooks) {
+  setupIntegrationTest(hooks);
+  test('it renders', async function(assert) {
+    this.set('array1', ['1', '2', '3', '4', '5']);
```

Comment:
array1, and array2 are not good variable names, it is not clear what is being tested fro this context.

Reply:
@CosmicCoder96 Understood, shall update it with better variable name